### DISCUSSION TITLE
V3 - Tenant

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#7419] [Help]
Hi, i test the v3 version of filament and the tenant system, i try to implement organizations for users :

Why i must implement the organization method belongsTo Organization ? In User list i see only the user with have orgnization_id for the current tenant

**Comment by pboivin** [comment]
Sorry what is the issue? It seems OK to only see the users for the current organization, no?

**Comment by warsemann** [comment]
I want the reverse :)
Users have many organization and i want to administrate them at the same place

**Reply by pboivin** [reply]
Ah ok, makes sense. Do you think a separate panel for this kind of "superadmin" access could work for you?

**Reply by FarhanShares** [reply]
As per doc, it says it automatically adds a global scope, what if I want to add a logic there to not add a global scope if it's a super admin, is there a way out? Can we not customize the scoping? 
https://filamentphp.com/docs/3.x/panels/tenancy#tenancy-security

**Reply by FarhanShares** [reply]
I got a solution in another discussion (attached the link below). Now I'll need to think how practical it would be. Should I go for this or create a separate panel!

https://github.com/filamentphp/filament/discussions/8059
